setup.py

```diff
@@ -19,6 +19,7 @@
     'qrcode',
     'pillow',
     'cryptography',
+    'typing',
```

Comment:
Changing the dependencies triggers a cascade of required changes, since all docker containers and RPM packages need to be updated also. Please have a look at the output of the containers. Probably best to avoid adding a new dependency for now. SInce all type annotations are string, I don't think you need to import 'typing', although pycharm might indicate differently.

Reply:
You should really make it a habit to run the containers on changes since then you can see what the impact is of your changes.

Reply:
A lot of the Travis builds were failing because of the missing typing module that's why I added it. But I will check if the build will succeed if I remove the module from setup.py. 
The only builds failing now are the pep8 and mypy builds, the pep8 is now fixed.
The mypy build is a bit more complicated, it gives a "misplaced type annotation" error and I don't know how to fix it.
![image](https://user-images.githubusercontent.com/36157555/63331328-c0646a00-c335-11e9-9cb6-0dff9b628dd5.png)


> You should really make it a habit to run the containers on changes since then you can see what the impact is of your changes.
Do you mean to check it with local containers?

Reply:
Arg documentation is restored. I also fixed the problem with mypy. Currently checking all of the files with pep8 and fixing all issues. I fixed the E501 error this way:
![image](https://user-images.githubusercontent.com/36157555/63335749-35d43880-c33e-11e9-86c3-1f2eeee84a67.png)
Is that okay?

Reply:
yes, that is okay. you can also add a comment to disable the E501 error checking in this case. In the end, it is all about readability and consistency but in this case, it doesn't really improve.